.eslintrc.json

```diff
@@ -4,5 +4,8 @@
     "plugin:react/recommended",
     "plugin:react-hooks/recommended",
     "@mintlify/eslint-config-typescript"
-  ]
+  ],
+  "rules": {
+    "react/prop-types": "off"
```

Comment:
What is this for?

Reply:
It disables a rule that checks for prop types, but since we are using typescript we don't use proptypes and don't need the rule.